.github/workflows/shellcheck.yml

```
@@ -0,0 +1,42 @@
+# Run ShellCheck on PRs and master.
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+
+permissions:
+  contents: read
+
+jobs:
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out this repository
+      uses: actions/checkout@v3
+
+    - name: Check out shared actions
+      uses: actions/checkout@v3
+      with:
+        repository: kdmccormick/openedx.github
+        #repository: openedx/.github
+        ref: kdmccormick/shellcheck-impl
```

```suggestion
        repository: openedx/.github
```
I will apply this change the PR after https://github.com/openedx/.github/pull/64/ merges.